arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -721,6 +725,8 @@ public virtual async Task<GetEdgeCollectionsResponse> GetEdgeCollectionsAsync(st
         /// <param name="documentId">The document ID of the vertex to update.</param>
         /// <param name="body"></param>
         /// <param name="query"></param>
+        /// <param name="serializationOptions">The serialization options. When the value is null the
```

Comment:
Maybe we should mention that the default serialization options apply when they are not provided here. Something like: "When the value is null the default options provided by the serializer are used."

The phrasing is the same as currently used elsewhere though, so not a big deal.